scripts/configure_dev.sh

```diff
@@ -61,6 +100,21 @@ elif [ "${OS}" = "darwin" ]; then
     install_or_upgrade automake
     install_or_upgrade shellcheck
     install_or_upgrade python3
+elif [ "${OS}" = "windows" ]; then
+    pacman -S --disable-download-timeout --noconfirm git automake autoconf m4 libtool mingw-w64-x86_64-python3 make mingw-w64-x86_64-gcc mingw-w64-x86_64-go mingw-w64-x86_64-boost mingw-w64-x86_64-python unzip procps
+    if [ $? -ne 0 ]
+    then
+        echo "Error installing pacman dependencies"
+        exit 1
+    fi
+
+    export GOPATH=$HOME/go
```

Comment:
Why configure_dev needs to export GOPATH here (only for windows)?

Reply:
I completely forgot about an old attempt to move all the actions of the `/scripts/windows` directory inside `configure_dev.sh` to normalize set up of development environment. I'll fix it.

About exporting `GOPATH`, because the whole process runs under some sort of virtual environment, I assume Go is not installed and I need it to run `configure_dev-deps.sh`, install shellcheck (there is no package available under MSYS2, and copy `libwinpthread-1.dll` (cannot compile statically because of `karalabe/hid` dependency using libusb)